HW7_OlhaMaraduda/HW7_Task3_Olha_Maraduda.py

```diff
@@ -0,0 +1,17 @@
+# Task3. Write a function that calculates the number of charactes in
+# given string.
+# input: "Hello"
+# output: {"h":1, "e":1, "l":2, "o":1}
+
```

Comment:
good idea
look at this version of code
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result